ruby_on_rails/README.md

```diff
@@ -1,45 +1,56 @@
-  - Controllers - [lesson](rails_basics/controller_basics.md)
-  - Views - [lesson](rails_basics/views.md)
-  - The Asset Pipeline - [lesson](rails_basics/asset_pipeline.md)
-  - Rails 7 Import Map - [lesson](rails_basics/importmap.md)
-  - Deployment - [lesson](rails_basics/deployment.md)
-  - [Blog App](rails_basics/project_blog_app.md)
-- Active Record Basics
-  - Active Record Basics - [lesson](databases_and_activerecord/active_record_basics.md)
-  - [Building With Active Record](databases_and_activerecord/project_ar_basics.md)
-- Forms and Authentication
-  - Form Basics - [lesson](forms_and_authentication/form_basics.md)
-  - [Forms](forms_and_authentication/project_forms.md)
-  - Sessions, Cookies, and Authentication - [lesson](forms_and_authentication/sessions_cookies_authentication.md)
-  - [Authentication](forms_and_authentication/project_auth.md)
-- Advanced Forms and Active Record
-  - Active Record Queries - [lesson](advanced_forms_and_activerecord/active_record_queries.md)
-  - Active Record Associations - [lesson](advanced_forms_and_activerecord/active_record_associations.md)
-  - [Associations](advanced_forms_and_activerecord/project_associations.md)
-  - Active Record Callbacks - [lesson](advanced_forms_and_activerecord/active_record_callbacks.md)
-  - Advanced Forms - [lesson](advanced_forms_and_activerecord/forms_advanced.md)
-  - [Building Advanced Forms](advanced_forms_and_activerecord/project_forms_advanced.md)
-- APIs
-  - APIs and Building Your Own - [lesson](apis/api_basics.md)
-  - Working With External APIs - [lesson](apis/api_interfacing.md)
-  - [Kittens API](apis/project_kittens_api.md)
-  - [Using an API](apis/project_using_an_api.md)
-- Mailers and Advanced Topics
-  - Mailers - [lesson](mailers_advanced_topics/mailers.md)
-  - [Sending Confirmation Emails](mailers_advanced_topics/project_mailers.md)
-  - Advanced Topics - [lesson](mailers_advanced_topics/advanced_topics.md)
-  - [Final Project](mailers_advanced_topics/project_final.md)
-  - Conclusion - [lesson](mailers_advanced_topics/conclusion.md)
-- Rails Sprinkles
-  - JS Bundling - [lesson](rails_sprinkles/js_bundling.md)
\ No newline at end of file
+The following list represents how the lessons are divided into sections and presented on the website.
+
+**Disclaimer:** Given the ever updating nature of the curriculum, the outline might be outdated. See the [Ruby on Rails course on the website](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails) instead.
+
+### Introduction
+1. [How this Course Will Work](introduction/how_this_course_will_work.md)
+2. [Preparing for Deployment](introduction/preparing_for_deployment.md)
+3. [**Project: Installing Rails**](introduction/project_installing_rails.md)
+4. [A Railsy Web Refresher](introduction/web_refresher.md)
+### Rails Basics
+1. [Routing](rails_basics/routing.md)
+2. [Controllers](rails_basics/controller_basics.md)
+3. [Views](rails_basics/views.md)
```

Comment:
The markdown file link for the Controllers lesson is broken. It needs to be fixed.

Reply:
I made a new commit to fix the issue with the Controllers lesson markdown file link. Please let me know if there is anything else to add/fix !